testing/test_terminal.py

```diff
@@ -506,6 +506,33 @@ def test_three():
         )
         assert result.ret == 0
 
+    def test_deselected_with_hookwrapper(self, testdir):
+        testpath = testdir.makeconftest(
+            """
+            def pytest_collection_modifyitems(config, items):
```

Comment:
This needs to be a hookwrapper probably.
Don't remember really now.

Reply:
I think the change is good regardless if it fixes a bug or not, btw.